taxcalc/tests/test_parameters.py

```diff
@@ -455,3 +455,14 @@ def test_parameters_get_default_start_year():
     assert meta_ctc_c['start_year'] == 2015
     assert meta_ctc_c['row_label'] == ["2015", "2016", "2017", "2018"]
     assert meta_ctc_c['value'] == [1000, 1000, 1000, 500]
+
+
+def test_params_to_json():
+    p = Parameters()
+    vals_as_json = p.params_to_json()
+    temporary_file = tempfile.NamedTemporaryFile(delete=False)
+    with open(temporary_file.name, 'w') as t:
+        t.write(vals_as_json)
+    p2 = Parameters(parameter_dict=json.load(open(temporary_file.name)))
+    assert json.loads(p2.params_to_json()) == json.loads(vals_as_json)
+    os.remove(temporary_file.name)
```

Comment:
Hmm.. if we throw an exception somewhere in here before reaching the `os.remove`, we will end up polluting the file system. What do you think about using temporary file test fixtures?